src/main/java/com/easypost/model/Error.java

```diff
@@ -59,22 +60,4 @@ public String getMessage() {
     public void setMessage(final String message) {
         this.message = message;
     }
-
-    /**
-     * Get the suggestion for the error.
-     *
-     * @return suggestion
-     */
-    public String getSuggestion() {
-        return suggestion;
-    }
-
-    /**
-     * Set the suggestion for the error.
-     *
-     * @param suggestion suggestion
-     */
-    public void setSuggestion(final String suggestion) {
-        this.suggestion = suggestion;
-    }
```

Comment:
I believe we still need `suggestion`. It's not often used but sometimes on address verification we'll actually suggest an update to the address to make it verify correctly. (search `suggestion` in our docs and you'll find it in the responses)

Reply:
Thanks for calling this out! Looks like I also need to bring `field` back as well